phoenix-core/src/main/java/org/apache/phoenix/monitoring/TableMetricsManager.java

```diff
@@ -287,4 +287,179 @@ public void clearTableLevelMetrics() {
     public void clear() {
         TableMetricsManager.clearTableLevelMetricsMethod();
     }
+
+    public static Map<String, List<HistogramDistribution>> getSizeHistogramsForAllTables() {
+        Map<String, List<HistogramDistribution>> map = new HashMap<>();
+        for (Map.Entry<String, TableClientMetrics> entry: tableClientMetricsMapping.entrySet()) {
+            TableHistograms tableHistograms = entry.getValue().getTableHistograms();
+            map.put(entry.getKey(), tableHistograms.getTableSizeHistogramsDistribution());
+        }
+        return map;
+    }
+
+    public static Map<String, List<HistogramDistribution>> getLatencyHistogramsForAllTables() {
+        Map<String, List<HistogramDistribution>> map = new HashMap<>();
+        for (Map.Entry<String, TableClientMetrics> entry: tableClientMetricsMapping.entrySet()) {
+            TableHistograms tableHistograms = entry.getValue().getTableHistograms();
+            map.put(entry.getKey(), tableHistograms.getTableLatencyHistogramsDistribution());
+        }
+        return map;
+    }
+
+    public static LatencyHistogram getUpsertLatencyHistogramForTable(String tableName) {
```

Comment:
A ton of these methods have the same patter of get the TableClientMetrics then null handle it... any way we can reduce the copy paste/boiler plate code here?